lib/prefab/options.rb

```diff
@@ -40,6 +42,7 @@ def initialize(
       shared_cache: NoopCache.new, # Something that quacks like Rails.cache ideally memcached
       namespace: "",
       log_formatter: DEFAULT_LOG_FORMATTER,
+      log_prefix: nil,
```

Comment:
Do we want to make the default be 'cloud.prefab.web'?

Reply:
I don't think so. It would really be 'com.customer.web'.  We can't really get the project name here, but that would be nice.  TBH I don't have a better idea than nil.